The d2to1 package is defunct and not offered by Void; roughly equivalent
functionality is provided by setuptools_scm, which is provided by Void. Drop
the special d2to1 setup call to allow setuptools_scm to work properly.

--- a/setup.py	2021-01-28 10:00:20.165871918 -0500
+++ b/setup.py	2021-01-28 10:01:07.760817952 -0500
@@ -58,17 +58,4 @@
 ## Normal d2to1 setup
 ##
 
-setup(
-    setup_requires=['d2to1'],
-    extras_require={
-        'Mustache': ["pystache", ],
-        'Mako': ["mako", ],
-        'test': [
-            "nose",
-            "minimock",
-            "mako",
-            "pystache",
-        ],
-    },
-    d2to1=True
-)
+setup(use_scm_version=True)
